pages/about/policies.html

```diff
@@ -66,10 +66,103 @@
                 </div>
             </div>
         </div>
-    </div>
-
+ 
     <br/>
-    <h1> Policies</h1>
+    <h1 class="ui block header center blue aligned"> School Policies</h1>
 
+        <div class="ui hidden section divider"></div>
+          <div class="ui stackable grid">
+            <div class="three column row">
+              <div class="column">
+                <h1 class="ui header">
+                  Child Protection
+                </h1>
+                <p>
+                  <b><i>Child Safeguarding Assessment of Risk of Mount Sion CBS Secondary School</i></b> In accordance with section 11 of the Children First Act 2015 and with the requirment of Chapter 8 of the <i>Child Protection Procedures for Primary and Post-Primary Schools 2017</i> the following is the written....
+                </p>
+                <div class="ui animated blue button" tabindex="0">
+                  <div class="visible content">View More</div>
+                  <div class="hidden content">
+                    <i class="right arrow icon"></i>
+                  </div>
+                </div>
+              </div>
+              <div class="column">
+                <h1 class="ui header">
+                  SPHE
+                </h1>
+                <p>
+                  <b><i>Social, Personal and Health Education: </i></b>"Mount Sion is a place where our children can develop physically, emotionally, intellectually and spiritually in a safe and caring environment". (Mission Statement)
+                  Mount Sion, CBS., Waterford is a Catholic all boys secondary....
+                </p>
+                 <div class="ui animated blue button" tabindex="0">
```

Comment:
Out of curiosity, what is the plan for the read more? External link, pop up, expanded tab?

Reply:
@ciaranRoche good question. part of the reason I stopped there. I though about adding a link to the documents but not sure that's appropriate. I think a separate page that is not linked in the nav with a copy of the policy is most appropriate. A little more difficult to update as it's hardcoded web content at that point. will leave a comment in the trello card to get some clarification on this.

Reply:
Is there a way where we can visibly see that it's an external link? If there is then we could use that for this page and other ones too.

Reply:
I think the view more should open up the appropriate website related to a grid if a person would like to read more about it.